ui/app/src/components/SystemStatus/GlobalDialogManager.tsx

```diff
@@ -62,7 +63,12 @@ function GlobalDialogManager() {
       {/* endregion */}
 
       {/* region Dependencies */}
-
+      <DependenciesDialog
+        open={state.dependencies.open}
+        item={state.dependencies.item}
+        dependenciesSelection={state.dependencies.dependenciesSelection}
```

Comment:
Please find a better prop name. `dependenciesSelection` is not clear what it is. Reach out if you want help.